production_validation_test.py

```
@@ -0,0 +1,340 @@
+
+        # ログから統計情報を抽出
+        for line in stdout.split('\n'):
+            if "返信要件分析完了" in line:
+                # 例: 総コメント数=22, 返信必要=4, 返信不要=18, 推定時間=286分
+                try:
+                    parts = line.split("総コメント数=")[1]
+                    stats["actionable_comments"] = int(parts.split(",")[0])
+                    stats["replies_required"] = int(parts.split("返信必要=")[1].split(",")[0])
+                    stats["replies_not_required"] = int(parts.split("返信不要=")[1].split(",")[0])
+                    stats["estimated_time"] = int(parts.split("推定時間=")[1].split("分")[0])
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)